shapes/contact/person.shex

```diff
@@ -22,8 +34,16 @@ vcard:Individual {
     vcard:role xsd:string ? ;
     vcard:title xsd:string ? ;
     vcard:note xsd:string ? ;
+    vcard:inAddressBook vcard:AddressBook + ;
+    # vcard:inAddressBook is a custom term
+    # vcard:AddressBook is a custom class
 }
 
+# We store the preferred WebID for each person. The use of vcard:WebId
+# refer to the Individual's WebIDs, but the idea is that we could expand it
+# to refer to other type of unique identifiers later
+# vcard:WebId is a custom term
+# Big question - SHOULD something as fundamental as WebID be defined in vCard?
 vcard:WebId {
```

Comment:
If we're going to introduce terms to vcard, maybe we should follow their casing convention?

I think that would mean: vcard:WebID instead of vcard:WebId.

Reply:
Yeah, I definitely free that the natural 'home' for WebID classes or properties is not really vCard (hence my big comment saying 'Big question - ...') on the line above! So I think this addition here in this ShEx is really just a conceptual suggestion at this point, and the details of the name casing would be discussed and decided as part of the bigger WebID discussion.